src/js/chrome/getusermedia.js

```
@@ -120,9 +120,13 @@ module.exports = function() {
   var shimError_ = function(e) {
     return {
       name: {
-        ConstraintNotSatisfiedError: 'OverconstrainedError',
         PermissionDeniedError: 'NotAllowedError',
-        TrackStartError: 'NotReadableError'
+        InvalidStateError: 'NotReadableError',
+        DevicesNotFoundError: 'NotFoundError',
+        ConstraintNotSatisfiedError: 'OverconstrainedError',
+        TrackStartError: 'NotReadableError',
```

@jan-ivar yes, Chrome 59 will (finally!) report an error for that instead of just returning with a track that fails later. Bug @ https://bugs.chromium.org/p/chromium/issues/detail?id=651910

(I wanted to rename TrackStartError there but... it turned out to be this rabbit hole :-))